### PR TITLE
[IMP] pos_order_cancel, pos_product_category_discount and other

### PR DESCRIPTION
**Pull request by GabbasovDinar**

### CODE REVIEW

**Comment by ilmir-k** [comment]
@GabbasovDinar fix lint error

**Comment by GabbasovDinar** [comment]
@ilmir-k DONE

File: pos_multi_session_restaurant/models/res_users_model.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+from odoo import fields
+from odoo import models
+
+
+class PosConfig(models.Model):
+    _inherit = 'res.users'
+
+    allow_decrease_kitchen_only = fields.Boolean('Allow change Qty for kitchen orders', default=False)
```
Comment (yelizariev) [inline pos_multi_session_restaurant/models/res_users_model.py]: I think it should be pos_disable_payment or pos_disable_payment_restaurant

**Review by yelizariev — CHANGES_REQUESTED** [review]

**Comment by yelizariev** [comment]
@GabbasovDinar Try to move ``allow_decrease_kitchen_only`` feature to ``pos_disable_payment`` without adding the restaurant dependency. It seems to be possible on js part. At the pos.config form, you can add invisible field ``is_restaurant_installed`` and show the field in that case only

File: pos_disable_payment/README.rst [hunk 2]
```diff
@@ -20,6 +20,7 @@ Contributors
 ------------
 * Ivan Yelizariev <yelizariev@it-projects.info>
 * Ilmir Karamov <karamov@it-projects.info>
+* Dinar Gabbasov <gabbasov@it-projects.info>
```
Comment (yelizariev) [inline pos_disable_payment/README.rst]: Replace emails to team links

**Comment by yelizariev** [comment]
@GabbasovDinar Prepare an issue with scenarios for tests

**Review by yelizariev — CHANGES_REQUESTED** [review]

**Comment by GabbasovDinar** [comment]
@yelizariev You can check the improvement for pos_disable_payment

File: pos_order_cancel/static/src/css/pos.css [hunk 3]
```diff
@@ -93,6 +93,12 @@
     background: black;
     border: 1px solid black;
 }
+.pos .orderlines .cancellation-info {
+    text-align: left;
+    font-size: 80%;
+    color: #ff6868;
+    margin-left: 13px;
```
Comment (ilmir-k) [inline pos_order_cancel/static/src/css/pos.css]: @GabbasovDinar I think we can remove `margin-left` from here or use more less value . Also add `margin-bottom: 3px;`

Comment (GabbasovDinar) [reply pos_order_cancel/static/src/css/pos.css]: @ilmir-k DONE

**Comment by ilmir-k** [comment]
@yelizariev your review is needed

File: pos_disable_payment/__openerp__.py [hunk 4]
```diff
@@ -2,13 +2,15 @@
...
-    'version': '2.4.0',
+    'version': '10.0.2.4.1',
     'author': 'IT-Projects LLC, Ivan Yelizariev',
     'license': 'LGPL-3',
     'category': 'Point Of Sale',
     "support": "apps@it-projects.info",
     'website': 'https://yelizariev.github.io',
-    'depends': ['point_of_sale'],
+    'depends': [
+        'pos_discount'
```
Comment (yelizariev) [inline pos_disable_payment/__openerp__.py]: Try without it

File: pos_disable_payment/static/src/js/pos_disable_payment.js [hunk 5]
```diff
@@ -88,26 +88,66 @@ odoo.define('pos_disable_payment', function(require){
...
+        },
+        check_kitchen_access: function(line){
+            var user = this.pos.cashier || this.pos.user;
+            var need_check = false;
+            if (!user.allow_decrease_amount) {
+                if (user.allow_decrease_kitchen_only) {
+                    return true;
+                }
+                need_check = true;
+            }
```
Comment (yelizariev) [inline pos_disable_payment/static/src/js/pos_disable_payment.js]: ``need_check`` is not needed to be checked.

```
if (user.allow_decrease_amount or user.allow_decrease_kitchen_only) {
return true;
}
```

**Review by yelizariev — CHANGES_REQUESTED** [review]

File: pos_disable_payment/static/src/js/pos_disable_payment.js (hunk 5 above)
Comment (yelizariev) [reply pos_disable_payment/static/src/js/pos_disable_payment.js]: original

Comment (yelizariev) [reply pos_disable_payment/static/src/js/pos_disable_payment.js]: original

**Review by ilmir-k — APPROVED** [review]
:+1:

**Comment by ilmir-k** [comment]
@yelizariev we need your approve

**Review by yelizariev — APPROVED** [review]

**Pull request merged**